data/hai/hai reveal 4 middle.txt

```diff
@@ -97,37 +96,37 @@ mission "Hai Reveal [A02] Symbol Identification"
+			`	Teeneep looks at you as if you were daft. "Kidnappers have no reason to search an uninhabited planet for potential victims. This gives us a safe place to hide critical people or information. If the kidnappings are a coordinated effort as we suspect, they would track down threats to the operation, like escaped victims, or our organization. We have plenty of space here to do our work without precautions that would be necessary on a populated planet."`
 				goto questions
 			label cadmium
-			`	A Hai in a medical uniform explains, "The victims each seem to have been exposed to airborne particulates containing cadmium which must be circulating through their ships ventilation. I should also clarify that it's not strictly 'poisoning'. The level of exposure is quite low, but enough that testing shortly after being rescued does show clearly elevated levels in many who were taken.`
-			`	"No known technology of human or Hai origin exhausts anything which could create this effect. There are some microbots used in Hai medicine that are used to seal off ruptures causing internal bleeding. Their dispenser canisters have a residue of residual cadmium dust in them, but those are the only thing we even bring onto a ship that uses it and are self-contained for disposal. The molecular bonding doesn't match anyway. There are some materials involved with properties unlike anything either of us make, which suggests this is coming from another race's technology."`
+			`	A Hai in a medical uniform explains, "The victims each have been exposed to complex cadmium compounds, shortly before rescue. These compounds leave a human or Hai quickly, but the level after rescue is high enough to detect.`
+			`	"No known human or Hai ship technology exhausts these cadmium compounds. There are some Hai medical microbots that leave behind similar compounds, but not the same ones, and those are self-contained for disposal. We believe this is coming from another race's technology."`
 			`	They share a glance with Teeneep, "We have our suspicions but they're not conclusive yet."`
 			branch questions
 				not "00 Hai Leaks Response 4B: declined"
-			`	"We did, however, get a hold of the scan results from the Geocoris you rescued. Interestingly, the self-repair systems it has appear to use a technology which could be responsible for something like this, and we suspect they came from the same place."`
+			`	"The Geodesic Geocoris you found has self-repair systems that use technology which could be responsible for something like this, and we suspect they came from the same place."`
```

Comment:
```suggestion
			`	"We were able to look at the scan of that modified Geocoris you recovered. It has self-repair systems that use technology which could be responsible for something like this, and we suspect they came from the same place."`
```

I think it was written so that the scan before meant something other than for a report.